shipment/buyer/overview.md

```diff
@@ -6,4 +6,6 @@ description: Overview of the buyer shipment process.
 
 ## Receive a shipment event
 
-As a buyer you can [Receive a shipment event](receive-shipment-event.md) when a shipment is new or updated.
+As a buyer you can [Receive a shipment event](receive/README.md) when a shipment is new or updated.
+
+Additionally you can [Download a document attached to a shipment](receive/download-document.md).
```

Comment:
The document doesn't get pushed ,right? it feels a bit weird to make this part of "Receive a shipment event"

Reply:
The attached documents are embedded in the shipment.
The customer needs this information to download the document attached to the shipment.
The customer can check if the shipment even contains new or updated documents to be downloaded.

Reply:
We use the words `receive` and `send` everywhere
`receive` means the information direction towards the customers
`send` means the information direction towards Tradecloud
This is regardless of push or pull.